src/components/new-request/search.vue

```diff
@@ -578,49 +595,47 @@ export default class Search extends Mixins(CommonMixin, NrAffiliationMixin) {
   }
 
   /** Called when Request Action menu item is changed. */
-  onRequestActionChange (request: RequestActionsI): void {
+  async onRequestActionChange (request: RequestActionsI): Promise<void> {
     this.request = request
 
-    // calculate whether to show tooltip
-    // (in next tick after DOM update)
-    Vue.nextTick(() => {
-      const el = document.querySelector('.request-action-select .v-select__selection') as any
-      const offsetWidth = el?.offsetWidth as number
-      const scrollWidth = el?.scrollWidth as number
-      this.showRequestActionTooltip = (offsetWidth < scrollWidth)
-    })
```

Comment:
Don't worry. I move this below, and I went with all-async in this method instead of using a callback in the nextTick function.